### PR TITLE
Versions up

### PR DESCRIPTION
**Pull request by brig**

### CODE REVIEW

**Comment by brig** [comment]
@concord-ci test it

**Comment by brig** [comment]
@concord-ci test it

File: k8s/agent-operator/pom.xml [hunk 1]
```diff
@@ -81,12 +81,6 @@
             <artifactId>jaxb-impl</artifactId>
         </dependency>
 
-        <!-- required by the fabric8 client, but not included automatically for some reason -->
```
Comment (ibodrov) [inline k8s/agent-operator/pom.xml:84]: I think the dependency is still needed, without it fabric8 fails during the init.

Comment (brig) [reply k8s/agent-operator/pom.xml:84]: oops

Comment (brig) [reply k8s/agent-operator/pom.xml:84]: dixed

**Comment by brig** [comment]
@concord-ci test it

**Comment by brig** [comment]
@concord-ci test it

**Comment by ibodrov** [comment]
@concord-ci test it

**Review by ibodrov — APPROVED** [review]

**Pull request merged**